posts/views/feed.py

```diff
@@ -15,8 +15,6 @@
 
 @feature_switch(features.PRIVATE_FEED, yes=require_auth, no=noop)
 def feed(request, post_type=POST_TYPE_ALL, room_slug=None, label_code=None, ordering=ORDERING_ACTIVITY):
-    post_type = post_type or Post
```

Comment:
This change is irrelevant to the PR.
Just randomly noticed the line that looks like a bug: `post_type` is string, `Post` is not. `posts.filter(type=Post)` below won't work with this default value

Reply:
Most likely a typo, thnx